tests/src/test/scala/cats/tests/ApplicativeTests.scala

```diff
@@ -4,7 +4,7 @@ package tests
 import cats.Applicative
 
 
-class ApplicativeTests extends CatsSuite {
+class ApplicativeCheck extends CatsSuite {
```

Comment:
Will do, good point. I didn't know about `MonadErrorSuite` or the rest of the `Check`-style suites.